src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpReplIdeFeatures.cs

```diff
@@ -0,0 +1,346 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.Shared.TestHooks;
+using Microsoft.VisualStudio.IntegrationTest.Utilities;
+using Roslyn.VisualStudio.IntegrationTests.Extensions.Interactive;
+using Xunit;
+
+namespace Roslyn.VisualStudio.IntegrationTests.CSharp
+{
+    [Collection(nameof(SharedIntegrationHostFixture))]
+    public class CSharpReplIdeFeatures : AbstractInteractiveWindowTest
+    {
+        public CSharpReplIdeFeatures(VisualStudioInstanceFactory instanceFactory)
+            : base(instanceFactory)
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(true);
+        }
+
+        public new void Dispose()
+        {
+            VisualStudioWorkspaceOutOfProc.SetUseSuggestionMode(false);
+            Reset();
+            base.Dispose();
+        }
+
+        // https://github.com/dotnet/roslyn/issues/801
+        // [Fact]
+        public void VerifyDefaultUsingStatements()
+        {
+            SubmitText("Console.WriteLine(42);");
+            this.WaitForLastReplOutput("42");
+        }
+
+        // https://github.com/dotnet/roslyn/issues/801
+        // [Fact]
+        public void VerifyCodeActionsAvailableInCurrentSubmission()
+        {
+            InsertCode("Directory.Exists(\"foo\");");
+            //      <VerifyCodeActions ApplyFix = "using System.IO;" >
```

Comment:
Code Actions are available in Interactive. If one opens Visual Studio, actions described in the original test can be performed.

However, the Visual Studio instance we use for tests does not support these code actions (set a break point during an integration test to see the difference).

So, I migrated this test (that was commented out in the original Tao test set) but have to keep it disabled. Please let me know if you see a better solution for this.